src/AudioPlugin/PluginProcessor.h

```diff
@@ -4,6 +4,8 @@
 
 #include <juce_audio_processors/juce_audio_processors.h>
 
+class AudioPluginAudioProcessorEditor;

```

Comment:
I'm very rusty in C++ but is this the same type as the class declared in https://github.com/msvbg/ml-vst/blob/69d0a24e5827b1fb0299bf4d076633153ac8dcbd/src/AudioPlugin/PluginEditor.h#L7 or can this redeclaration introduce subtle problems?

Reply:
This is a "forward-declaration", which I use to avoid unnecessary `#include`s or circular `#include`s. I don't think it should cause any problems, but am also rusty